CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+## Contributing to KubeVirt Tekton Tasks
+
+### Our workflow
+
+Contributing to KubeVirt Tekton Tasks should be as simple as possible. Have a question? Want
+to discuss something? Want to contribute something? Just open an
+[Issue](https://github.com/kubevirt/kubevirt-tekton-tasks/issues), a [Pull
+Request](https://github.com/kubevirt/kubevirt-tekton-tasks/pulls), or send a mail to our
+[Google Group](https://groups.google.com/forum/#!forum/kubevirt-dev).
+
+If you spot a bug or want to change something pretty simple, just go
+ahead and open an Issue and/or a Pull Request, including your changes
+at [kubevirt/kubevirt-tekton-tasks](https://github.com/kubevirt/kubevirt-tekton-tasks).
+
+For bigger changes, please create a tracker Issue, describing what you want to
```

Comment:
I'm not a fan of requiring an issue; I prefer each PR to explain everything about itself. But you're the boss of this repo.

Reply:
I don't mind either, I mostly copied this from kubevirt repo. Altough, I guess an advantage of this is that you can discuss potential new features/tasks before people start working on them.

I suppose for people who would be in sync with this repo activity, this should not be a requirement.